tests/e2e/support/index.ts

```diff
@@ -108,7 +108,7 @@ const customCommands: CustomCommands = {
       cy.typeQuery(query).type('{enter}');
     });
   },
-  typeQuery: query => cy.getByDataTest('search-input').type(query),
+  typeQuery: query => cy.getByDataTest('search-input').type(query, { force: true }),
```

Comment:
needed but not desirable. 
Seems to be working without forcing it in newer Cypress versions https://github.com/cypress-io/cypress/issues/21433